v20_0_prerelease/location.js

```diff
@@ -78,14 +82,9 @@ class FormItMap {
                             this._updatePushPin();
                             this._focusLocation();
                         });
-                        //no results found by auto-suggest API, so try to geocode with exactly what the user gave us.
                     } else {
-                        this._address = this._addressInput.value;
-                        this._location = undefined;
-                        this._geocodeLocationAddress(() => {
-                            this._updatePushPin();
-                            this._focusLocation();
```

Comment:
@garrisb82 can you describe the change here so @deanstein can understand the UX consequences? In this case we won't update the push pin or focus the location?

Reply:
Brett has demo'd the changes for me and Helena and we liked what we saw.

Reply:
@matturday the push pin still updates, but that is done in the _setAddress call this was all replaced with.